opm/autodiff/AquiferCarterTracy.hpp

```diff
@@ -164,65 +71,30 @@ namespace Opm
                 pitd_prime = Opm::linearInterpolationDerivative(aquct_data_.td, aquct_data_.pi, td);
             }
 
-            inline void initQuantities(const Aquancon::AquanconOutput& connection)
-            {
-                // We reset the cumulative flux at the start of any simulation, so, W_flux = 0
-                W_flux_ = 0.;
-
-                // We next get our connections to the aquifer and initialize these quantities using the initialize_connections function
-                initializeConnections(connection);
-
-                calculateAquiferCondition();
-
-                calculateAquiferConstants();
```

Comment:
calculateAquiferConstants() is not called in the `initQuantities()` of AquiferInterface. 

which causes Tc_ is zero and that is how the NaN is generated. 

I mean, calculateAquiferConstants() is called in the `initQuantities` of the old `AquiferCarterTracy` class.

Reply:
in my opinion, the easiest way to fix this is to also to create a function `calculateAquiferConstants()` for the `AquiferFetkovich` to update `Tc_` too. Then you add `calculateAquiferConstants()` in the `initQuantities()` of the base class `AquiferInterface`. 

It is just a suggestion and I am not the one to guarantee the correctness of the algorithms here.

Reply:
Right, Thanks! I believe I just fixed it :-) The 2D_CTAQUIFER case now also runs smoothly. I think we are ready for jenkins